runtime/compiler/trj9/control/J9Recompilation.cpp

```diff
@@ -173,6 +173,12 @@ J9::Recompilation::findOrCreateProfileInfo()
       profileInfo = new (PERSISTENT_NEW) TR_PersistentProfileInfo(DEFAULT_PROFILING_FREQUENCY, DEFAULT_PROFILING_COUNT);
       _methodInfo->setRecentProfileInfo(profileInfo);
       _bodyInfo->setProfileInfo(profileInfo);
+
```

Comment:
Please add some stats regarding the footprint taken by the TR_PersistentInfo. Decrement when deleting the information. Print in CompilationThread.cpp when TR_PrintCompMem env var is set.

Reply:
I have added footprint profiling for `TR_PersistentProfileInfo` to the JProfiling thread. However, I don't believe this is worthwhile, as the bulk of the memory overhead is due to `TR_ValueProfileInfo` and `TR_BlockFrequencyInfo`. I'd be happy to add instrumentation to these data structures to evaluate their footprint, but I'd prefer to do this in a separate PR, if that's okay with you.